consensus/spos/bls/blsSubroundsFactory_test.go

```diff
@@ -431,6 +449,28 @@ func TestFactory_NewFactoryNilSignaturesTrackerShouldFail(t *testing.T) {
 	container := mock.InitConsensusCore()
 	worker := initWorker()
 
+	fct, err := bls.NewSubroundsFactory(
+		container,
+		consensusState,
+		worker,
+		chainID,
+		currentPid,
+		&statusHandler.AppStatusHandlerStub{},
+		&testscommon.SentSignatureTrackerStub{},
+		nil,
+	)
+
+	assert.Nil(t, fct)
+	assert.Equal(t, spos.ErrNilThrottler, err)
+}
+
+func TestFactory_NewFactoryNilThrottlerShouldFail(t *testing.T) {
```

Comment:
the test name for this function and the one from above have to swapped

Reply:
solved